analytics-samples/kotlin-sample/build.gradle

```diff
@@ -0,0 +1,83 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+apply from: rootProject.file('gradle/android.gradle')
+
+def WEBHOOK_AUTH_USERNAME = ""
```

Comment:
do we need all these configs? i am thinking there may be some bloat from the other sample app